tests/Unit/IO/Test_H5.cpp

```diff
@@ -469,13 +468,19 @@ SPECTRE_TEST_CASE("Unit.IO.H5.ReadData", "[Unit][IO][H5]") {
                                  h5::AccessType::ReadWrite);
   const hid_t group_id = my_group.id();
 
+  // Test writing and reading a DataVector as a rank-1 dataset
+  const DataVector rank1_data{1.0, 2.0, 3.0};
```

Comment:
you shadowed two variable names below with `rank1_data` and `rank1_data_from_file`

Reply:
Done :)